test/sensitivities/linalg/strided.jl

```diff
@@ -1,9 +1,20 @@
 @testset "Strided" begin
-
+    RS = StridedMatrix{<:∇Scalar}
+    RST = Transpose{<:∇Scalar, RS}
+    RSA = Adjoint{<:∇Scalar, RS}
+    strided_matmul_combinations = (
+        (RS,  RS,  'N', 'C', :Ȳ, :B, 'C', 'N', :A, :Ȳ),
+        (RST, RS,  'N', 'T', :B, :Ȳ, 'N', 'N', :A, :Ȳ),
+        (RS,  RST, 'N', 'N', :Ȳ, :B, 'T', 'N', :Ȳ, :A),
+        (RST, RST, 'T', 'T', :B, :Ȳ, 'T', 'T', :Ȳ, :A),
+        (RSA, RS,  'N', 'C', :B, :Ȳ, 'N', 'N', :A, :Ȳ),
+        (RS,  RSA, 'N', 'N', :Ȳ, :B, 'C', 'N', :Ȳ, :A),
+        (RSA, RSA, 'C', 'C', :B, :Ȳ, 'C', 'C', :Ȳ, :A),
+    )
+    # TODO: This test seems like it doesn't actually test the combinations.
```

Comment:
It really doesn't. But I didn't write these tests.

Reply:
I am tempted to delete this file, i know these are all tested properly in ChainRules.jl